One test currently failing: 
https://github.com/Chaste/ApPredict/actions/runs/4357003508/jobs/7615762345

```
8/22 Test #543: TestMetadataCellmlModels ................***Failed    0.25 sec
Running 1 test

 ***** TestMetadataCellmlModels.hpp *****
Entering TestMetadataHasBeenCorrectlyTranslatedToChaste
Model 1: shannon_wang_puglisi_weber_bers_2004_model_updated

/home/runner/_work/ApPredict/ApPredict/Chaste/projects/ApPredict/test/TestMetadataCellmlModels.hpp:333: Error: Expected (HasAParameterForThisOrScalingFactorOfThis(p_chaste_cell, "membrane_transient_outward_current_conductance") == true), found (false != true)
/home/runner/_work/ApPredict/ApPredict/Chaste/projects/ApPredict/test/TestMetadataCellmlModels.hpp:175: Error: Test failed: 
Chaste error: ./ode/src/common/AbstractOdeSystemInformation.cpp:143: No parameter named 'membrane_transient_outward_current_conductance'.
Failed

Failed 1 of 1 test
Success rate: 0%

...

The following tests FAILED:
	543 - TestMetadataCellmlModels (Failed)
```

--- a/test/TestMetadataCellmlModels.hpp
+++ b/test/TestMetadataCellmlModels.hpp
@@ -323,7 +323,7 @@ public:
             const double default_d = GetParameterOrScalingFactorForThis(p_chaste_cell, "membrane_slow_delayed_rectifier_potassium_current_conductance");
 
             double default_e = 0;
-            if (model_index == 7u || model_index == 4u)
+            if (model_index == 7u || model_index == 4u || model_index == 1u)
             {
                 TS_ASSERT_EQUALS(HasAParameterForThisOrScalingFactorOfThis(p_chaste_cell, "membrane_fast_transient_outward_current_conductance"), true);
                 default_e = GetParameterOrScalingFactorForThis(p_chaste_cell, "membrane_fast_transient_outward_current_conductance");
@@ -347,7 +347,7 @@ public:
                 SetParameterOrScalingFactorOfThis(p_chaste_cell, "membrane_rapid_delayed_rectifier_potassium_current_conductance", default_c * c);
                 SetParameterOrScalingFactorOfThis(p_chaste_cell, "membrane_slow_delayed_rectifier_potassium_current_conductance", default_d * d);
 
-                if (model_index == 7u || model_index == 4u)
+                if (model_index == 7u || model_index == 4u || model_index == 1u)
                 {
                     SetParameterOrScalingFactorOfThis(p_chaste_cell, "membrane_fast_transient_outward_current_conductance", default_e * e);
                 }
@@ -371,7 +371,7 @@ public:
                 SetParameterOrScalingFactorOfThis(p_chaste_cell_opt, "membrane_L_type_calcium_current_conductance", default_b * b);
                 SetParameterOrScalingFactorOfThis(p_chaste_cell_opt, "membrane_rapid_delayed_rectifier_potassium_current_conductance", default_c * c);
                 SetParameterOrScalingFactorOfThis(p_chaste_cell_opt, "membrane_slow_delayed_rectifier_potassium_current_conductance", default_d * d);
-                if (model_index == 7u || model_index == 4u)
+                if (model_index == 7u || model_index == 4u || model_index == 1u)
                 {
                     SetParameterOrScalingFactorOfThis(p_chaste_cell_opt, "membrane_fast_transient_outward_current_conductance", default_e * e);
                 }
@@ -395,7 +395,7 @@ public:
                 SetParameterOrScalingFactorOfThis(p_cvode_cell, "membrane_L_type_calcium_current_conductance", default_b * b);
                 SetParameterOrScalingFactorOfThis(p_cvode_cell, "membrane_rapid_delayed_rectifier_potassium_current_conductance", default_c * c);
                 SetParameterOrScalingFactorOfThis(p_cvode_cell, "membrane_slow_delayed_rectifier_potassium_current_conductance", default_d * d);
-                if (model_index == 7u || model_index == 4u)
+                if (model_index == 7u || model_index == 4u || model_index == 1u)
                 {
                     SetParameterOrScalingFactorOfThis(p_cvode_cell, "membrane_fast_transient_outward_current_conductance", default_e * e);
                 }
@@ -419,7 +419,7 @@ public:
                 SetParameterOrScalingFactorOfThis(p_cvode_cell_opt, "membrane_L_type_calcium_current_conductance", default_b * b);
                 SetParameterOrScalingFactorOfThis(p_cvode_cell_opt, "membrane_rapid_delayed_rectifier_potassium_current_conductance", default_c * c);
                 SetParameterOrScalingFactorOfThis(p_cvode_cell_opt, "membrane_slow_delayed_rectifier_potassium_current_conductance", default_d * d);
-                if (model_index == 7u || model_index == 4u)
+                if (model_index == 7u || model_index == 4u || model_index == 1u)
                 {
                     SetParameterOrScalingFactorOfThis(p_cvode_cell_opt, "membrane_fast_transient_outward_current_conductance", default_e * e);
                 }

